tests/component/system/test_device_properties.py

```diff
@@ -112,3 +112,70 @@ def test___module___get_chassis___shared_interpreter(device: Device):
     chassis = device.compact_daq_chassis_device
 
     assert chassis._interpreter is device._interpreter
+
+
+def test___ext_cal_last_date_and_time___no_errors(any_x_series_device: Device) -> None:
+    last_date_and_time = any_x_series_device.ext_cal_last_date_and_time
+
+    assert last_date_and_time is not None
+
+
+def test___self_cal_last_date_and_time___no_errors(any_x_series_device: Device) -> None:
+    last_date_and_time = any_x_series_device.self_cal_last_date_and_time
+
+    assert last_date_and_time is not None
+
+
+def test___device_supports_cal___no_errors(any_x_series_device: Device) -> None:
+    is_cal_supported = any_x_series_device.device_supports_cal
+
+    assert is_cal_supported is True
+
+
+def test___cal_acc_connection_count___no_errors(any_x_series_device: Device) -> None:
+    test_value = 99
+
+    try:
+        current_acc_connection_count = any_x_series_device.cal_acc_connection_count
+
+        any_x_series_device.cal_acc_connection_count = test_value
+        assert test_value == any_x_series_device.cal_acc_connection_count
+
+        any_x_series_device.cal_acc_connection_count = current_acc_connection_count
+
+    except DaqError as e:
+        if e.error_code != DAQmxErrors.ATTR_NOT_SUPPORTED:
```

Comment:
Use pytest.raises for error tests, something like this. 
```
with pytest.raises(DaqException) as exc_info:
   _ = any_x_series_device.cal_acc_connection_count

assert exc_info.value.error_code == DAQmxErrors.ATTR_NOT_SUPPORTED
```

Since this is an SC Express specific property and you are testing with X Series, you can assume the attribute always errors.

Reply:
Noted, test case is updated as per suggestion.